docs/development/e2e-testing.md

```
@@ -0,0 +1,460 @@
+kubectl --context mgmt1 get dpa -n openshift-adp
+kubectl --context mgmt2 get dpa -n openshift-adp
+```
+
+## Comprehensive Test Scenarios
+
+### Scenario 1: Complete Switchover Workflow
+
+**Objective**: Validate full primary→secondary switchover with all 3 managed clusters
+
+**Test Matrix:**
+| Method | Old Hub Action | Primary Context | Secondary Context | Expected Outcome |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)